static/app/views/performance/transactionSummary/transactionTags/tagsDisplay.tsx

```diff
@@ -45,7 +144,8 @@ const TagsDisplay = (props: Props) => {
             tagKeyLimit={HISTOGRAM_TAG_KEY_LIMIT}
             numBucketsPerKey={HISTOGRAM_BUCKET_LIMIT}
             tagKey={tagKey}
-            sort="-frequency"
+            cursor={cursor}
+            sort={tagSort}
```

Comment:
This can be an empty list right so no sort will be enforced. Should there be a default sort on frequency in that case?

Reply:
It was already handled inside the query itself, but I'm going to move it up so it's explicit 👍